python/chemiscope/explore.py

```diff
@@ -30,6 +30,12 @@ def explore(frames, featurize=None, properties=None, mode="default"):
         with the atomic structures. Properties can be extracted from frames with
         :py:func:`extract_properties` or manually defined by the user.
 
+    :param environments: optional. List of environments (described as
+        ``(structure id, center id, cutoff)``) to include when extracting the
+        atomic properties. Can be extracted from frames with
+        :py:func:`all_atomic_environments` (or :py:func:`librascal_atomic_environments`)
```

Comment:
```suggestion
        :py:func:`all_atomic_environments`.
```

I think we can stop promoting `librascal_atomic_environments` now. I'll see about removing it altogether.